odoo/tests/common.py

```diff
@@ -1004,6 +1004,10 @@ def get_http_request_threads():
             self._logger.info('remaining requests')
             odoo.tools.misc.dumpstacks()
 
+    def logout(self, keep_db=True):
+        self.session.logout(keep_db=True)
+        odoo.http.root.session_store.save(self.session)
```

Comment:
can you refresh my memory about this last line ?

Reply:
I need to logout the user to test if the pages visited after are tracked properly. I don't think there is another way to do so.

Reply:
nice to have a param keep_db unsued.
why not use it in logout() call to...

But btw, why can't you use the real logout.

If we patch 80% of the methods, we will only test 20% of the code ...

Reply:
What do you mean by real Logout ? In an HTTPCase I don't have any current request so I have to do it manually.